cms/templates/manage_users.html

```diff
@@ -94,14 +94,15 @@ <h3 class="title">${_('Add Team Members to This Course')}</h3>
     <aside class="content-supplementary" role="complementary">
       <div class="bit">
         <h3 class="title-3">${_("Course Team Roles")}</h3>
-        <p>${_("Course team members, or staff, are course co-authors. They have full writing and editing privileges on all course content.")}</p>
+        <p>${_("Course team members with the Staff role are course co-authors. They have full writing and editing privileges on all course content.")}</p>
         <p>${_("Admins are course team members who can add and remove other course team members.")}</p>
+        <p>${_("All course team members are automatically enrolled in the course and can access content in Studio, the LMS, and Insights.")}</p>
       </div>
 
       % if show_transfer_ownership_hint:
       <div class="bit">
         <h3 class="title-3">${_("Transferring Ownership")}</h3>
-        <p>${_("Every course must have an Admin. If you're the Admin and you want transfer ownership of the course, click Add admin access to make another user the Admin, then ask that user to remove you from the Course Team list.")}</p>
+        <p>${_("Every course must have an Admin. If you are the Admin and you want to transfer ownership of the course, click Add admin access to make another user the Admin, then ask that user to remove you from the Course Team list.")}</p>
```

Comment:
is that a standard we plan to apply to the UI as well as in our docs? I think that if we decide that it is a standard we want to apply, we should make the effort to go through and apply it uniformly and all at once, unlike the incremental changes we're making to the docs.